### PR TITLE
Add no_upscale and rotate filters

### PR DESCRIPTION
**Pull request by gitanuj**

### CODE REVIEW

File: src/main/java/com/squareup/pollexor/ThumborUrlBuilder.java [hunk 1]
```diff
@@ -788,5 +790,23 @@ public static String blur(int radius, int sigma) {
...
+  }
+
+  /**
+   * This filter rotates the given image according to the angle passed.
+   * @param angle The angle of rotation. Values can be either 0°, 90°, 180° or 270° – multiples of
+   *              90°. Angles equal to or greater than 360° will be replaced by their coterminal
+   *              angle of rotation.
+   */
+  public static String rotate(int angle) {
+    if (angle % 90 > 0) {
```
Comment (JakeWharton) [inline src/main/java/com/squareup/pollexor/ThumborUrlBuilder.java]: This should be `!= 0`.

**Comment by JakeWharton** [comment]
Looks good. Just one comment.

**Comment by gitanuj** [comment]
Done.

**Comment by JakeWharton** [comment]
Thanks! Squashed and merged to `master`.

**Pull request closed without merging**